bin/configurer/setup-local-idam.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -eu
+
+dir=$(dirname ${0})
+
+# Defaults are ok and create require ccd_gateway service in idam
+#${dir}/utils/create-ccd-service.sh
```

Comment:
Is that needed at any point?

Reply:
Yes, I have uncommented it's the initial service we need, refactor a bit though.